src/bin/index.js

```diff
@@ -108,12 +119,15 @@ yargs(hideBin(process.argv))
           'Whether to automatically configure the deploy alias compatible with the lightweight Mina blockchain network.',
```

Comment:
```suggestion
          'Whether to automatically configure the deploy alias to be compatible with the lightweight Mina blockchain network.',
```

Reply:
if we mean that using the `lightnet` parameter makes the deploy alias compatible with Lightnet

Reply:
https://github.com/o1-labs/zkapp-cli/issues/592